web/js/containers/sidebar/sidebar.js

```diff
@@ -246,14 +247,22 @@ class Sidebar extends React.Component {
       };
 
     return (
-      <a
-        href={embedWVLogoLink}
-        title={WVLogoTitle}
-        id="wv-logo"
-        className={isDistractionFreeModeActive ? 'wv-logo-distraction-free-mode' : ''}
-        style={sidebarStyle}
-        onClick={(e) => this.handleWorldviewLogoClick(e, permalink)}
-      />
+      !isKioskModeActive ? (
```

Comment:
We need to remove the NOT here (i.e. `isKioskModeActive ? `) & then this looks good to me.